### ISSUE TITLE
Option `port` is required on Electron.

### ISSUE THREAD

**Issue by xland** [webtorrent/webtorrent#1783]
Electron 7.1.2 and 6.1.4
Windows 10

Code:
```js
    var WebTorrent = require("webtorrent");
    if (WebTorrent.WEBRTC_SUPPORT) {
      console.log(1);
    } else {
      console.log(2);
    }
    var client = new WebTorrent();
    var torrentId =
      "magnet:?xt=urn:btih:08ada5a7a6183aae1e09d831df6748d566095a10&dn=Sintel&tr=udp%3A%2F%2Fexplodie.org%3A6969&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Ftracker.empire-js.us%3A1337&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2F&xs=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel.torrent";
    client.add(torrentId, function(torrent) {
      var file = torrent.files.find(function(file) {
        return file.name.endsWith(".mp4");
      });
      file.appendTo(".home");
    });
```

Error:
> index.js?17f3:13 Uncaught Error: Option `port` is required
    at new Discovery (index.js?17f3:13)
    at Torrent._startDiscovery (torrent.js?c115:317)
    at Torrent._onListening (torrent.js?c115:288)
    at Torrent._onParsedTorrent (torrent.js?c115:244)
    at eval (torrent.js?c115:205)
    at processTicksAndRejections (internal/process/task_queues.js:75)

I try these code in the browser. It is ok.

**Comment by xland** [comment]
@alxhotel 

How about this issue's progressing?

**Comment by alxhotel** [comment]
Hi @xland,

I just tested this in Electron 10.1.1 and it works without an error.

Please update your Electron version and let me know if it works.

**Comment by rimday** [comment]
this issue appear at new version, you can temporary fix this by use version 0.108.6
